### DISCUSSION TITLE
runtime memory usage vs gtk4 in basic empty window

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [hecrj/iced#1603] [Q&A]
Hi. Is there any benchmark regarding iced vs. gtk4 in resource management?

**Comment by 13r0ck** [comment]
Not scientific in anyway, and the results are biased towards iced, because Gtk has some features like accessibility that Iced doesn't which will inherently add some negative performance to iced, but in my experience, the same apps after being rewritten in iced use 1/2 to 1/3 the memory usage. Redraw time is faster too, but I have no explicit data for that.
